open_dread_rando/files/custom_scenario.lua

```diff
@@ -54,6 +54,14 @@ function Scenario.InitScenario(arg1, arg2, arg3, arg4)
             Init.sThisRandoIdentifier, tostring(currentSaveRandoIdentifier)
     ))
 
+    -- This function is added via an exefs patch. It not existing means either broken install or unknown executable
+    if not Game.HasRandomizerPatches then
+        return Scenario.ShowFatalErrorMessage({
+            "{c2}Error!{c0}|Unsupported Metroid Dread version.",
+            "Only {c6}1.0.0{c0}, {c6}2.0.0{c0} and {c6}2.1.0{c0} are supported.|Returning to title screen.",
```

Comment:
I would actually just say 1.0.0 and 2.1.0. 2.0.0 was only supported because it was current at the time